src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  margin: 0;
+  padding: 0;
```

Comment:
You drop the paddings and margins for the whole `body` in the styles, do you really have to do it again here?